fs_attachment/models/ir_attachment.py

```diff
@@ -374,13 +376,22 @@ def _set_attachment_data(self, asbytes) -> None:  # pylint: disable=missing-retu
     def _storage_file_read(self, fname: str) -> bytes | None:
         """Read the file from the filesystem storage"""
         fs, _storage, fname = self._fs_parse_store_fname(fname)
-        try:
-            with fs.open(fname, "rb") as f:
-                return f.read()
-        except IOError:
-            _logger.info(
-                "Error reading %s on storage %s", fname, _storage, exc_info=True
-            )
+        for attempt in range(FILE_READ_RETRIES):
+            try:
+                with fs.open(fname, "rb") as f:
+                    return f.read()
+            except FileNotFoundError:
+                _logger.info(
```

Comment:
The number of attempt and the delay could be properties of the `fs.storage` record.  It could make no sense to have retry mechanism on others file systems than s3

Reply:
Yes, I totally agree so I did the change.